design/design-doc-stacks.md

```diff
@@ -56,17 +56,19 @@ All further functionality for Crossplane (databases, buckets, etc.) could then b
 This section describes the end to end installation flow implemented by the Stack Manager:
 
 * The SM starts up with a default “source” registry (e.g. `registry.crossplane.io`) that contains packages (bundles of a Stack and its custom controllers and CRDs) published to it
-* User creates an `ExtensionRequest` instance to request a Stack be installed in the cluster, which includes everything needed to successfully run that Stack.  The `ExtensionRequest` includes:
```

Comment:
we should update the example yaml for `ExtensionRequest` to show the new `PlatformStackRequest` and `ApplicationStackRequest` types instead.